middleware/modelExport.js

```diff
@@ -298,7 +328,18 @@ module.exports = {
         asyncFn.push(helpers.getForm);
         asyncFn.push(helpers.getGrid);
 
-        async.waterfall(asyncFn, function (err, filters, form, grid) {
+        // get the actual export object
```

Comment:
I think the multi exports (with stage 2 implemented) will give them the ability to write their own if the default export don't meet their needs.

I'm assuming you can define an export button without having to use the default Linz export. I'm basing my assumption on your example DSL below:

```
    grid: {
        export: [
            {
                label: 'Custom export',
                action: 'url/endpoint'
            }
        ]
    }
```